src/renderer/ScreenController.tsx

```diff
@@ -129,6 +130,9 @@ export default class ScreenController extends Component {
     if (this.selectedAccountId === undefined) {
       return
     }
+
+    this.unselectChatRef.current?.()
```

Comment:
What does this mean here? Looks weird to me :)

Reply:
The function is set from inside [`ChatContext`](https://github.com/deltachat/deltachat-desktop/pull/3789/files#diff-e9605783d283da9708373aa54111f85308a01ee55595a58be274bb20f34000a7). Read it as `chatContext.unselectChat()`

Reply:
An alternative to doing it like that is to introduce the `unselectChat` as a prop to the classic class-style React component:

https://github.com/deltachat/deltachat-desktop/blob/adz/i3776-fix-rerendering/src/renderer/ScreenController.tsx#L330-L337